### PR TITLE
Parminder theme

### PR DESCRIPTION
**Pull request by Parm-S**
Adding AppBar

### CODE REVIEW

File: src/components/ThemeSetter.tsx [hunk 1]
```diff
@@ -1,31 +1,34 @@
-import { createTheme, ThemeProvider } from "@mui/material/styles";
-import React, { useState } from "react";
+import React from "react";
+import {
```
Comment (nik-1207) [inline src/components/ThemeSetter.tsx:2]: should be in single line

File: src/components/ThemeSetter.tsx [hunk 2]
```diff
@@ -1,31 +1,34 @@
...
-const ThemeSetter: React.FC = (props) => {
-  //    Read current Theme from local storage
-  const currentThemeName = localStorage.getItem("appTheme") || "lightTheme";
-  //    state to hold the selected theme Name
-  const [themeName, _setThemeName] = useState(currentThemeName);
+const APPTHEME = "appTheme";
+
+const ThemeProvider: React.FC = (props) => {
+  const currentThemeName = localStorage.getItem(APPTHEME) || "lightTheme";
+
```
Comment (nik-1207) [inline src/components/ThemeSetter.tsx:14]: unnecessary empty line

File: src/components/Header/Header.tsx [hunk 3]
```diff
@@ -1,28 +1,43 @@
-import Button from "@mui/material/Button";
 import React from "react";
+import AppBar from "@mui/material/AppBar";
+import Box from "@mui/material/Box";
+import Toolbar from "@mui/material/Toolbar";
+import Typography from "@mui/material/Typography";
+import Button from "@mui/material/Button";
+import Brightness2OutlinedIcon from "@mui/icons-material/Brightness2Outlined";
+import Brightness7OutlinedIcon from "@mui/icons-material/Brightness7Outlined";
 import { useContext } from "react";
```
Comment (nik-1207) [inline src/components/Header/Header.tsx]: should be after line 1

File: .husky/_/husky.sh [hunk 4]
```diff
@@ -29,3 +29,4 @@ if [ -z "$husky_skip_init" ]; then
 
   exit $exitCode
 fi
+
```
Comment (AbhayMaheshwari123) [inline .husky/_/husky.sh]: unnecessary change

File: src/components/Header/Header.tsx (hunk 3 above)
Comment (AbhayMaheshwari123) [reply src/components/Header/Header.tsx]: or just add in the first line itself

File: src/index.tsx [hunk 5]
```diff
@@ -3,10 +3,13 @@ import ReactDOM from "react-dom";
 import "./index.css";
```
Comment (nik-1207) [inline src/index.tsx]: if you are using mui why index.css

**Review by AbhayMaheshwari123 — COMMENTED** [review]
small changes to do

**Review by nik-1207 — COMMENTED** [review]
please check

File: src/components/ThemeSetter.tsx (hunk 1 above)
Comment (Parm-S) [reply src/components/ThemeSetter.tsx:2]: prettier do it

File: src/components/ThemeSetter.tsx (hunk 2 above)
Comment (Parm-S) [reply src/components/ThemeSetter.tsx:14]: okay

File: src/components/Header/Header.tsx (hunk 3 above)
Comment (Parm-S) [reply src/components/Header/Header.tsx]: okay

File: src/index.tsx (hunk 5 above)
Comment (Parm-S) [reply src/index.tsx]: okay

File: .husky/_/husky.sh [hunk 6]
```diff
@@ -29,4 +29,3 @@ if [ -z "$husky_skip_init" ]; then
 
   exit $exitCode
 fi
-
```
Comment (nik-1207) [inline .husky/_/husky.sh]: why>

File: package.json [hunk 7]
```diff
@@ -8,6 +8,7 @@
     "@mui/icons-material": "^5.0.5",
     "@mui/material": "^5.0.6",
     "@mui/styled-engine-sc": "^5.0.4",
+    "@mui/styles": "^5.0.2",
     "@testing-library/jest-dom": "^5.11.4",
```
Comment (nik-1207) [inline package.json]: test dependency should be in dev

File: public/index.html [hunk 8]
```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Tambola | Number-Generator</title>
```
Comment (nik-1207) [inline public/index.html]: no requirement of NUMBER-GENERTATOR

File: src/App.tsx [hunk 9]
```diff
@@ -1,12 +1,25 @@
+import { Theme } from "@mui/material";
+import { makeStyles } from "@mui/styles";
 import React from "react";
```
Comment (nik-1207) [inline src/App.tsx]: react on top

File: src/App.tsx [hunk 10]
```diff
@@ -1,12 +1,25 @@
...
-import "./App.css";
 import { Header } from "./components/Header/Header";
 /**
```
Comment (nik-1207) [inline src/App.tsx]: remove it

**Review by nik-1207 — COMMENTED** [review]
check requested changes

File: src/components/Header/Header.tsx [hunk 11]
```diff
@@ -6,11 +6,23 @@ import Typography from "@mui/material/Typography";
 import Button from "@mui/material/Button";
 import Brightness2OutlinedIcon from "@mui/icons-material/Brightness2Outlined";
 import Brightness7OutlinedIcon from "@mui/icons-material/Brightness7Outlined";
-import { useContext } from "react";
```
Comment (AbhayMaheshwari123) [inline src/components/Header/Header.tsx]: should be on top

**Review by AbhayMaheshwari123 — COMMENTED** [review]
check once

File: src/App.tsx (hunk 10 above)
Comment (Parm-S) [reply src/App.tsx]: okay

File: .husky/_/husky.sh (hunk 6 above)
Comment (Parm-S) [reply .husky/_/husky.sh]: done by prettier

File: package.json (hunk 7 above)
Comment (Parm-S) [reply package.json]: done

File: public/index.html (hunk 8 above)
Comment (Parm-S) [reply public/index.html]: okay

File: src/App.tsx (hunk 9 above)
Comment (Parm-S) [reply src/App.tsx]: done

**Review by AbhayMaheshwari123 — APPROVED** [review]
